enterprise/frontend/src/embedding-sdk/components/private/AppInitializeController.tsx

```diff
@@ -23,14 +26,33 @@ export const AppInitializeController = ({
   useInitData({ config });
 
   const isInitialized = useSdkSelector(getIsInitialized);
+  const licenseProblem = useSdkLicenseProblem(config);
+
+  const content = useMemo(() => {
+    if (!isInitialized) {
+      return <SdkLoader />;
+    }
+
+    if (licenseProblem?.severity === "error") {
+      return null;
+    }
```

Comment:
We make the components underneath the `MetabaseProvider` not render if it encountered a license / usage error, mainly to prevent misuse or abuse of SDK.

Reply:
If they put the provider as wrapper of the entire app, this will make the entire customer's app break if they have an issue with the license

Reply:
@npretto fixed in commit https://github.com/metabase/metabase/pull/47821/commits/fd208075e24658a7718571f24d9d4ddc814cc616 where it only makes the SDK components not load, as opposed to the whole page. Thanks!